samples/java-retrofit/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Mar 04 14:07:30 EET 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=http\://services.gradle.org/distributions/gradle-2.2-bin.zip
```

Comment:
There should not be 2 gradle wrappers in one project. It also should not be necessary.
You can use `gradlew` from the root.